application/static/swagger.yml

```diff
@@ -0,0 +1,291 @@
+openapi: 3.0.0
+info:
+  title: Auctionable Change
+  description: Backend API for the Auctionable Change mobile application.
+                Created by Turing School of Software and Design students
+                Stephanie Friend, Kevin McGrevey, Andrew Reid.
+  version: 1.0.0
+  authors: Stephanie Friend, Kevin McGrevey, Andrew Reid
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+servers:
+  - url: https://auctionable-change-api.herokuapp.com
+    description: Main (production) server
+  - url: http://127.0.0.1:5000
+    description: Local (development) server
+tags:
+  - name: Items
+    description: Endpoints for Item CRUD
+  - name: Bids
+    description: Endpoints for Bid CRUD
+  - name: Charities
+    description: Endpoints for querying Charity Navigator
+paths:
+  /items:
+    get:
+      summary: Returns a list of available items.
+      tags:
+        - Items
+      responses:
+        '200':    # status code
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Item'
+
+        '400':
+          description: Bad Request
+    post:
+      summary: Creates a new item.
+      tags:
+        - Items
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Item_create'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Item'
+        '400':
+          description: Bad Request
+  /items/<id>:
+    get:
+      summary: Returns a selected items.
+      tags:
```

Comment:
Forgive me, but I'm going to be super picky. Is there anyway to update this to singular 'item'? So it reads 'Returns a selected item.'

Reply:
Good catch. Sloppy copy/paste job on my part.